modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path

```diff
@@ -67,6 +67,11 @@
 		<td>//div[@class='dropdown-menu autocomplete-dropdown-menu show']//button[@class='dropdown-item lfr-object__single-select--with-label' and contains(., '${key_entryName}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>CONDITION_SECTION_TITLE</td>
+		<td>//h3[(@class='lfr-objects__card-title') and text()='Condition']</td>
+		<td></td>
+	</tr>
```

Comment:
Can you change this to be more general? I see a few paths that exist with specific paths instead of using a variable for _text()_.
 This way people won't keep making new paths for very specific elements. 
Eventually, we would need to change out the instances that use the specific paths and use the new one you submit.